backend/src/main/java/com/htecgroup/skynest/model/response/BucketResponse.java

```diff
@@ -0,0 +1,18 @@
+package com.htecgroup.skynest.model.response;
+
+import com.htecgroup.skynest.model.entity.CompanyEntity;
+import com.htecgroup.skynest.model.entity.UserEntity;
+import lombok.Data;
+
+@Data
+public class BucketResponse {
+  private UserEntity createdBy;
```

Comment:
should we send entire entities in responses?

or just emails/ids

Reply:
As I wrote in the other PR, entities should never be exposed over the network.. create separate model and add just the necessary attributes

Reply:
You're right, I'll make the change